src/app.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<title>MovieGallery</title>
+		<meta charset="utf-8" />
+		<link rel="icon" href="$lib/image/Nav_Bar_Logo_For_StarWars_Franchise.jpeg" />
```

Comment:
Does this work? (`$lib/path/to/image`)

Reply:
Nope it didn't work i added a real favicon now!

https://github.com/BowenGeense/MovieShowcase/pull/3/files#diff-c279397c4feb4b0067eda16dfafc3453d18caa9afcc11de4d368411fefc03601